base/applications/regedit/lang/bg-BG.rc

```diff
@@ -661,8 +661,8 @@ STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS | WS_B
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Област за износ", IDC_STATIC, 2, 0, 366, 48
-    CONTROL "Вси&чки", IDC_EXPORT_ALL, "Button", BS_AUTORADIOBUTTON, 10, 10, 50, 11
-    CONTROL "Из&брания клон", IDC_EXPORT_BRANCH, "Button", BS_AUTORADIOBUTTON, 10, 22, 100, 11
+    AUTORADIOBUTTON "Вси&чки", IDC_EXPORT_ALL, 10, 10, 99, 11, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Из&брания клон", IDC_EXPORT_BRANCH, 10, 22, 99, 11
```

Comment:
99 ? why?

Reply:
I wrote that in the PRs description already: The upper number for IDC_EXPORT_ALL that was chosen initially for en_US was too small for the vast majority of languages, so most other langs introduced extrawursts of increasing that value historically (and inconsistently). We do have a lot of space available here to enlarge for all languages, so I could have actually used the 100 also from IDC_EXPORT_BRANCH (it makes sense to keep both AUTORADIOBUTTONs the same width).
I opted for 99 instead of 100 for both, because that shrinks the source-lines-length by 1char which might be of use for adding actual translations, or adding STYLES.
I considered the 99 being future-proof enough, so I preferred to stay out of the 3-digit-land.